todo_app/data/trello_items.py

```diff
@@ -1,68 +1,70 @@
-    url = f"https://api.trello.com/1/boards/{board_id}/lists?key={api_key}&token={api_token}"
-    response = requests.get(url)
-    data = response.json()
+    def find_list_id_by_name(self, board_id, name):
+        url = f"https://api.trello.com/1/boards/{board_id}/lists?key={self.api_key}&token={self.api_token}"
+        response = requests.get(url)
+        data = response.json()
     
-    for item in data:
-        if item['name'] == name:
-            return item['id']
-    return None
+        for item in data:
+            if item['name'] == name:
+                return item['id']
+        return None
 
-def find_card_id_by_name(list_id, card_name, api_key, api_token):
-    url = f"https://api.trello.com/1/lists/{list_id}/cards?key={api_key}&token={api_token}"
-    response = requests.get(url)
-    data = response.json()
+    def find_card_id_by_name(self, list_id, card_name):
+        url = f"https://api.trello.com/1/lists/{list_id}/cards?key={self.api_key}&token={self.api_token}"
+        response = requests.get(url)
+        data = response.json()
     
-    for item in data:
-        if item['name'] == card_name:
-            print (card_name)
-            return item['id']
-    return None
+        for item in data:
+            if item['name'] == card_name:
+                print(card_name)
+                return item['id']
+        return None
 
-def update_card_to_done(done_list_id, card_id, api_key, api_token):
-    # Trello API endpoint for updating a card's properties, such as moving it to a different list
-    url = furl = f"https://api.trello.com/1/cards/{card_id}?key={api_key}&token={api_token}"
+    def update_card_to_done(self, done_list_id, card_id):
+        # Trello API endpoint for updating a card's properties, such as moving it to a different list
+        url = f"https://api.trello.com/1/cards/{card_id}"
 
-    params = {
-        'key': api_key,       
-        'token': api_token,   
-        'idList': done_list_id 
-    }
+        params = {
+            'key': self.api_key,       
+            'token': self.api_token,   
+            'idList': done_list_id 
+        }
     
-    try:
-        # Perform the API call using POST method to update the card
-        response = requests.post(url, params=params)
+        try:
+            # Perform the API call using PUT method to update the card
+            response = requests.put(url, params=params)
 
-        return True
+            return True
     
-    except requests.exceptions.RequestException as e:
-        # Handle exceptions that may occur during the API call, such as network errors
-        print("Error occurred during API call:", e)
-        return False
\ No newline at end of file
+        except requests.exceptions.RequestException as e:
+            # Handle exceptions that may occur during the API call, such as network errors
+            print("Error occurred during API call:", e)
+            return False
```

Comment:
Minor, but worth noting that this only captures _exceptions_ in the request - which unfortunately doesn't include _failed_ requests. So maybe having the `try` block return `response.ok` which is a property that states whether or not the request succeeded would make sense

Reply:
Updated, thanks again.